.ci/dist-tag.groovy

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = "synthetics"
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    NPMRC_SECRET = 'secret/jenkins-ci/npmjs/elasticmachine'
+    TOTP_SECRET = 'totp/code/npmjs-elasticmachine'
+    NPM_PACKAGE = '@elastic/synthetics'
+    SLACK_CHANNEL = '#synthetics-user_experience-uptime'
+  }
+  options {
```

Comment:
if no concurrency is allowed then I'd say to use the below option:
- https://www.jenkins.io/doc/book/pipeline/syntax/#available-options

```suggestion
  options {
    disableConcurrentBuilds()
```

Reply:
AFAIK, concurrency on this Jenkins job should not be a problem?